target/linux/x86/image/Makefile

```diff
@@ -1,5 +1,6 @@
 # 
 # Copyright (C) 2006-2012 OpenWrt.org
+# Copyright (C) 2019 Alexander Couzens <lynxis@fe80.eu>
```

Comment:
While at it, you could add all authors:

```
git log --format="%aN <%aE>" target/linux/x86/image/Makefile | sort -u
```

Really, we should get rid of this nonsense, not encourage it. We've VCS for authorship records.

Reply:
If @lynxis don't mind I remove the line